### PR TITLE
Monica mooney

### PR DESCRIPTION
**Pull request by monmooney5523**
First PR for Final Project.

Feature: as the game begins, objects are randomly added to the board. The quantity and location of these objects vary. The objective of the game is for the player to move around the board, collecting as many objects as they can. After 10 seconds, these objects are removed and new objects are placed. 

Please provide any comments / feedback you feel is necessary! Thanks in advance.

### CODE REVIEW

**Comment by borisv1307** [comment]
All checks have failed.

**Comment by monmooney5523** [comment]
> All checks have failed.

Updated the use of Random to instead use SecureRandom(). Also updated the time between random object generation and refactored code. 

All tests are passing, final submission.

File: src/main/java/wrappers/RandomWrapper.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    }
+
+    public Point getRandomPassableTile() {
+        ArrayList<Point> passableTiles = getAllPassableTiles();
+        int numberOfPassableTiles = passableTiles.size();
+        if (numberOfPassableTiles > 0) {
+            int randomIndex = getRandomIntInRange(numberOfPassableTiles);
+            return passableTiles.get(randomIndex);
+        } else {
+            return null;
```
Comment (borisv1307) [inline src/main/java/wrappers/RandomWrapper.java:31]: NEVER RETURN NULL
Never allow null
Throw a new exception if you have to.

**Review by borisv1307 — CHANGES_REQUESTED** [review]

File: src/main/java/engine/GameEngine.java [hunk 2]
```diff
@@ -176,7 +177,7 @@ public void removePreviouslyAddedObjects() {
 
     public void refreshCollectableObjects() {
         int randomNumberOfObjects = randomWrapper.getRandomNumberOfObjects();
-        ArrayList<Point> tilesToBeUpdated = getPassableTiles(randomNumberOfObjects);
+        List<Point> tilesToBeUpdated = getTilesForObjects(randomNumberOfObjects);
         if (collectableObjects != null) {
```
Comment (borisv1307) [inline src/main/java/engine/GameEngine.java:181]: Remove this check and redesign to make sure it's never null.
Initialize it to an empty list in the constructor, for example.

**Review by borisv1307 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**